plugins/mongodb_logs.yaml

```
@@ -61,13 +72,13 @@ template: |
         # {"t":{"$date":"2022-04-26T16:15:44.876+00:00"},"s":"I",  "c":"INDEX",    "id":20345,   "ctx":"LogicalSessionCacheRefresh","msg":"Index build: done building","attr":{"buildUUID":null,"namespace":"config.system.sessions","index":"lsidTTLIndex","commitTimestamp":null}}
         - id: 4_4_parser
           type: json_parser
-          parse_from: body
+          parse_to: body
           timestamp:
-            parse_from: attributes.t.$date
+            parse_from: body.t.$date
```

Looks like this issue was always present, but I think the env-var replacement is biting us here.
```suggestion
            parse_from: body.t.$$date
```

---

I'm confused I jsut ran the plugin and it appeared to parse correctly? 

```
LogRecord #99
ObservedTimestamp: 2022-09-08 17:54:33.026062 +0000 UTC
Timestamp: 2022-09-08 17:53:37.549 +0000 UTC
Severity: I
Body: {
     -> component: STRING(CONTROL)
     -> context: STRING(initandlisten)
     -> message: STRING(db version v4.0.25)
     -> raw_log: STRING(2022-09-08T17:53:37.549+0000 I CONTROL  [initandlisten] db version v4.0.25)
     -> severity: STRING(I)
     -> timestamp: STRING(2022-09-08T17:53:37.549+0000)
}
Attributes:
     -> log.file.name: STRING(test.log)
     -> log_type: STRING(mongodb)
Trace ID:
Span ID:
Flags: 0
```

---

Yeah just verified it's always broken I'll work on fixing it.

---

The double `$$` works. I'll update and retest.

---

Ahh now i see the issue nvm me 🙃